src/sdk/namespace_fs.js

```diff
@@ -1759,6 +1761,13 @@ class NamespaceFS {
             throw new RpcError('BAD_REQUEST', 'Bad Request');
         }
     }
+
+    _skip_version_dir(dir_key) {
+        if(dir_key.indexOf(`${HIDDEN_VERSIONS_PATH}`) >= 0) {
```

Comment:
we expect at least 1 char before that so I think the condition should be > 0

Reply:
For `.versions` under bucket path will have 0 index. Ex: for  `$BUCKET_PATH/.versions` , the `dir_key` would be `.versions` that will have 0 index.

correct me if wrong

Reply:
ah ok coool